Update README (#46)
grafana/otel-checker: README.md

# OTel Me If It's Right

> Checker for if the implementation of OpenTelemetry instrumentation is correct by scanning the code in your repository.

Reviewer: I realize that this is not all - we also try to make a call to the endpoint using your api token: https://github.com/grafana/otel-checker/blob/main/checks/grafana/grafanaChecker.go#L113

> Checker for if the implementation of OpenTelemetry instrumentation is correct by scanning the code in your repository.

Reviewer 2: ```suggestion
This is a *checker* that analyzes if your implementation of OpenTelemetry instrumentation is correct.
```

or something on those lines, without the "by scanning the code in your repository"

Author: OK cool - I'll address Gregor's comment, then we can expand it later.

> Checker for if the implementation of OpenTelemetry instrumentation is correct by scanning the code in your repository.

Author: ```suggestion
Checker for if the implementation of OpenTelemetry instrumentation is correct by scanning the code in your repository, checking environment variables, validating your Grafana token and more.
```
